.github/workflows/py_checks.yml

```
@@ -151,3 +151,16 @@ jobs:
       - name: Run Bandit
         run: python -m bandit -r ./ -f screen
         working-directory: src/bindings/python/src/compatibility/openvino
+
+        # Python API 2.0 tests Flake code-style
```

I'd prefer to change also this comment
```suggestion
        # layer_tests Flake code-style
```